Wireframe/index.html

```diff
@@ -8,25 +8,54 @@
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
+    <h1>Wireframe to Code</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        Simple sketches to coding the application.
       </p>
     </header>
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img src="./ReadMe.webp" alt="picture of a note with readme sign" />
+        <h2>What is the purpose of the README file?</h2>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+          
```

Comment:
I think that the text inside of this `p` tag can be split in separate paragraphs.